program_info/win_install.nsi.in

```diff
@@ -149,6 +149,8 @@ Section "@Launcher_CommonName@"
     WriteRegStr HKCU "${UNINST_KEY}" "DisplayVersion" "@Launcher_VERSION_NAME@"
     WriteRegStr HKCU "${UNINST_KEY}" "VersionMajor" "@Launcher_VERSION_MAJOR@"
     WriteRegStr HKCU "${UNINST_KEY}" "VersionMinor" "@Launcher_VERSION_MINOR@"
+    WriteRegStr HKCR ".zip" "" "MyApp"
```

Comment:
Should the key values really be "MyApp"?

Reply:
Fixed by you :p

Reply:
fixed with my changes included